index.js

```diff
@@ -0,0 +1,7 @@
+window.addEventListener('DOMContentLoaded', () => {
+  const btn = document.querySelector('#menu_btn');
+const menu = document.querySelector('#mobile-nav');
```

Comment:
- [ ]  According to the project requirements the menu should disappear when we click on the close X button. As I mentioned above after adding the close button please add a click event to that close button, So the menu should disappear when the users click on the X button.